pkgs/os-specific/linux/powertop-unstable/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gettext
+, libnl
+, libtraceevent
+, libtracefs
+, ncurses
+, pciutils
+, pkg-config
+, zlib
+, autoreconfHook
+, autoconf-archive
+, nix-update-script
+, testers
+, powertop
+}:
+
+stdenv.mkDerivation rec {
+  pname = "powertop-unstable";
```

Comment:
Wait, what's this?

Reply:
To differentiate from https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/linux/powertop/default.nix